_search-plugins/concurrent-segment-search.md

```diff
@@ -52,6 +83,35 @@ PUT <index-name>/_settings
 ```
 {% include copy-curl.html %}
 
+
+When evaluating whether concurrent segment search is enabled on a cluster, the `search.concurrent_segment_search.mode` setting takes precedence over `search.concurrent_segment_search.enabled` setting.
+If the `search.concurrent_segment_search.mode` is not explicitly set, the `search.concurrent_segment_search.enabled` setting will be evaluated to determine whether to enable concurrent segment search.
+
+For clusters upgrading from an older version with the existing `search.concurrent_segment_search.enabled` setting, this setting will continue to be honored. However, once the `search.concurrent_segment_search.mode` is set, it will override the previous setting, enabling or disabling concurrent search based on the specified mode.
```

Comment:
```suggestion
When upgrading a cluster from an earlier version that specifies the older `search.concurrent_segment_search.enabled` setting, this setting will continue to be honored. However, once the `search.concurrent_segment_search.mode` is set, it will override the previous setting, enabling or disabling concurrent search based on the specified mode.
```